.github/workflows/enhanced_gradients-publish.yml

```diff
@@ -22,15 +22,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Set up Dart
-        uses: dart-lang/setup-dart@v1
-        with:
-          sdk: 3.0.0
-
       - name: Set up Flutter
```

Comment:
@bartekpacia do you know if this was needed? I am pretty sure that installing flutter installs dart as well

Reply:
TL;DR publishing doesn't work without it

See https://github.com/leancodepl/mobile-tools/blob/0a67ef6c49933a560fb3f6633465ce83032dfdf8/.github/actions/pub-release/action.yaml#L74-L76 for more context.